pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -140,6 +144,12 @@ func init() {
 	prometheus.MustRegister(pushTimeouts)
 	prometheus.MustRegister(pushes)
 	prometheus.MustRegister(pushErrors)
+
+	// Experimental env to disable push suppression
+	// By default a pod will not receive a push from an older version if a
+	// push for a newer version has started. This behavior can be disabled
+	// for testing or in special cases (bugs or corner cases found, etc)
+	suppressPush = os.Getenv("PILOT_SKIP_SUPPRESS") == ""
```

Comment:
Allow_concurrent_push

Reply:
done

Reply:
Regarding charts - will change in separate PR, want to test the /ready a bit more.